### PR TITLE
[Config] Logs - Add default logs related parameters

### PR DESCRIPTION
**Pull request by tmichelet**
So we play well with config files and env variables (docker)

Notes:

- only `log_enable` is an official parameter. Other parameters are for internal usage, and are not documented
- `run_path` is not great, doesn't work on all platforms, etc. This is a temporary workaround for logs,  which are linux only. We will make this platform agnostic when we work on windows support. Also, this is used only because we need a path on filesystem to store a file. If there is an existing configuration parameter we can use, please let me know. Alternative would be to rename `defaultJMXPipePath` to something more generic

I'll add a link to related log-agent PR when ready

### CODE REVIEW

**Comment by tmichelet** [comment]
cc @masci , @ajacquemot

File: pkg/config/config.go
Comment (olivielpeau) [inline pkg/config/config.go]: I think we should go with the alternative you mentioned: `defaultJMXPipePath` is not used anymore so let's remove it, instead we should have a `defaultRunPath` that points to `/opt/datadog-agent/run/` on unix by default (not sure about Windows, feel free to add a `FIXME` for windows).

The logs agent can then use that path.

**Review by olivielpeau — DISMISSED** [review]
one comment on the run path, other than that LGTM 👍

**Comment by tmichelet** [comment]
@olivielpeau updated. Also, let's please not merge before I open the second PR for log-agent repo

**Review by olivielpeau — APPROVED** [review]
LGTM

**Comment by tmichelet** [comment]
see also https://github.com/DataDog/datadog-log-agent/pull/16

**Comment by tmichelet** [comment]
@olivielpeau I think we're ready to merge, PR on log-agent side is ready. I'll test as soon as merged and release on agent-dev docker hub

**Review by ajacquemot — APPROVED** [review]
👍

**Pull request merged**
